manifests/install.pp

```diff
@@ -15,14 +15,27 @@
   }
   #TODO Configure Postfix properly, maybe as satellites and have a mail server to handle all our sent mail?
 
-~>file { '/etc/network/interfaces':
-    ensure  => 'present',
-    content => template('proxmox/interfaces.erb'),
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644'
+  network::interface { 'vmbr0':
+    family       => 'inet',
+    ipaddress    => $::ipaddress,
+    netmask      => $::netmask,
+    gateway      => $gatewayv4,
+    bridge_ports => [ $::netdev ],
+    bridge_stp   => 'off',
+    bridge_fd    => 0,
   }
-  reboot { '/etc/network/interfaces':
-    subscribe => [ File['/etc/network/interfaces'], Package['proxmox-ve'] ],
```

Comment:
Maybe just subscribe to the proxmox-ve package. The ordering shoud make sure it happens after the network config.

Reply:
Resolved by https://github.com/SineQuaNonSoftware/puppet-proxmox/pull/7/commits/67658a5faa077d46e715658a86407109aad1d146